### DISCUSSION TITLE
A Ghost Codespace prebuild is accumulating storage without it existing.

### DISCUSSION THREAD

**Discussion by YR3** [community/community#61610] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

**Hello Evreyone,**

I have disabled my Codespaces pre-builds and deleted all the old codespace prebuild actions runs couple of months in my forked repo and I've even fully deleted it Deleted on May 30. I am surprised why my consumption continuously starts to increase even though I am not using any such services. Because of this, I am unable to use Codespaces because Github says that I have used 100% of my Codespace usage (through wrongly billed Codespace storage). this really rendered the service useless, and disturbed lots of other projects that I was planning to spin-up.

<details><summary>Details</summary>
<p>

`2023-07-17,Codespaces - Linux,Prebuild storage,1.0416,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
2023-07-18,Codespaces - Linux,Prebuild storage,1.0416,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
2023-07-19,Codespaces - Linux,Prebuild storage,1.0416,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
2023-07-20,Codespaces - Linux,Prebuild storage,1.0416,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
2023-07-21,Codespaces - Linux,Prebuild storage,1.0416,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
2023-07-22,Codespaces - Linux,Prebuild storage,1.0416,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
2023-07-23,Codespaces - Linux,Prebuild storage,1.0416,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
2023-07-24,Codespaces - Linux,Prebuild storage,0.0868,gb-month,0.07,1.0,YR3,Ghost-fork,,Create Codespaces Prebuilds,
`
</p>
</details> 

I've checked the usage report and it's the same old now deleted repository, I remember deleting it to stop it from accumulating storage but it's still phantom-accumulating storage.

I tried recreating the repo with the exact name, then I've spun up a Codespace so I could disable it then delete the repo, but that didn't work.

It looks like this is a bug similar to
https://github.com/orgs/community/discussions/49291
and:
https://github.com/orgs/community/discussions/48213#discussioncomment-5081863

Is there any way this bug can be looked into by @Github_staff from the backend when they have time?

_PS: I'm pretty sure the fact that the repo's name being "Ghost-fork" has nothing to do with it acting like a haunting ghost._

**Many Thanks,**
**YR3**

**Comment by ErcSmi** [comment]
Same+1

**Comment by craiglpeters** [comment]
@YR3 we are investigating and will respond shortly

**Reply by YR3** [reply]
I guess somebody manually sorted things out backend-side, Thanks 🙏🏼

**Comment by daeh** [comment]
This is happening for me as well.

**Comment by petchged** [comment]
So do I. I have been having this issue for a few months now.

**Comment by IMGROOT2** [comment]
Same, this has been happening for a few months. And now, when I need to use Codespaces, I can't anymore. Please fix this ASAP!

**Comment by aniketvp24** [comment]
Same issue with me, I have been not been using codespaces and still my usage has reached 100%.

**Comment by adntgv** [comment]
same

**Comment by ghost** [comment]
Hi. I have same issue. I had deleted all my codespaces a year ago and my prebuild storage gets full every time

**Comment by petchged** [comment]
I still have the same issue.  I am not sure who to contact.

Sent from Samsung Galaxy smartphone.
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by ghost** [reply]
I found a solution here [https://github.com/orgs/community/discussions/49291#discussioncomment-5840075](https://github.com/orgs/community/discussions/49291#discussioncomment-5840075)

**Reply by YR3** [reply, marked as answer]
Could be it, I had this option ticked before Deleting it, so the lesson is:
never have the `Reduce prebuild available to only specific regions` on and remove all configuration before Codespace deletion?
